drivers/SmartThings/zigbee-switch/src/aqara-light/init.lua

```diff
@@ -33,17 +33,18 @@ local function do_refresh(self, device)
 end
 
 local function do_configure(self, device)
-  device:send(ColorControl.commands.MoveToColorTemperature(device, 200, 0x0000))
   device:configure()
-  do_refresh(self, device)
-end
-
-local function device_added(driver, device, event)
   device:send(cluster_base.write_manufacturer_specific_attribute(device,
     PRIVATE_CLUSTER_ID, PRIVATE_ATTRIBUTE_ID, MFG_CODE, data_types.Uint8, 1)) -- private
 
   device:send(Level.attributes.OnTransitionTime:write(device, 0))
   device:send(Level.attributes.OffTransitionTime:write(device, 0))
+  device:send(ColorControl.commands.MoveToColorTemperature(device, 200, 0x0000))
+
+  do_refresh(self, device)
+end
+
+local function device_added(driver, device, event)
 end
```

Comment:
I think you should just be able to delete this function (and in the handlers map below). It doesn't look like the base driver defines an `added` handler so you shouldn't need this no-op to overwrite it.